riot_api/methods/create_summoners_list.py

```diff
@@ -19,89 +13,23 @@ def create_summoner_list(players_list: list, server_id: str):
     """
     try:
         # dictionary for data input
-        members_to_add = {
-            server_id: [],
-        }
+        members_to_add = []
 
         # accessing each player
-        for summoner in players_list:
-
-            user = watcher.summoner.by_name(MY_REGION, summoner)
-
-            # First check if we have existing record for given summoner name
-            # Create query; TODO: check for update_time
-            summoner_cached_query = session.query(Summoners).filter(
-                Summoners.summoner_name == user["name"]
-            )
-
-            # TODO: Retrieve from db.
-            # Execute query
-            summoner_cached = summoner_cached_query.first()
-
-            if summoner_cached:
-                print("summoner found!")
-                # summoner_cached = dict(summoner_cached.__dict__)
-
-            user_name = user["name"]
-
-            ranked_stat = watcher.league.by_summoner(MY_REGION, user["id"])
+        for user_input_list_name in user_input_list_names:
+            # 'get_summoner_rank' will handle getting summoner's data.
+            summoner_data = get_summoner_rank(user_input_list_name)
```

Comment:
@jung1411 You didn't need to do all of other stuff. `get_summoner_rank()` gets you what you needed. 
You just needed to think deeper and make adjustments so this can be used in other functions as well.